src/utils/borrowed_games.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+export default class BorrowedGames {
+  async getBorrowedGames() {
+    return await axios
+      .get("https://api.chimaera.games/api/borrowedgames")
```

Comment:
La url la usan mucho qué pasaría si decidiéramos usar una versión de stage?

Lo mejor en esos casos es tirar de variable al estilo:
```js
const basePath='https://api.chimaera.games/api';
```

Y luego ir concatenando.

```suggestion
      .get(`${basePath}/borrowedgames`)
```

Es más axios tiene un modelo diseñado para esto.